src/components/common/Tag/Tag.tsx

```diff
@@ -0,0 +1,31 @@
+import { size } from "lodash-es";
+import { P5, Tag as TagUI, lightTheme, Stack } from "@deskpro/deskpro-ui";
+import type { FC } from "react";
+
+const Tag: FC<{ tag: string }> = ({ tag }) => (
+  <TagUI
+    color={{
+      borderColor: lightTheme.colors.grey80,
+      backgroundColor: `${lightTheme.colors.grey80}33`,
+      textColor: lightTheme.colors.grey80,
+    }}
+    label={tag}
+    withClose={false}
+  />
+);
+
+const Tags: FC<{ tags: string[] }> = ({ tags }) => {
+  if (!Array.isArray(tags) || !size(tags)) {
```

Comment:
No need for `lodash` here, `tags.length === 0` will work here.

Reply:
removed